src/models/connections.ts

```diff
@@ -93,6 +93,40 @@ const connections = (sequelize: Sequelize) => {
         return result.map((r: any) => r.toJSON() as ConnectionModel);
     };
 
+    const findAllFollowersByName = async (
+        name: string,
+        offset = 0,
+        limit = 20,
+        order: 'DESC' | 'ASC' = 'DESC'
+    ): Promise<ConnectionModel[]> => {
+        let result = await model.findAll({
+            attributes: ['creator'],
+            where: { name },
+            offset,
+            limit,
+            order: [['createdAt', order]],
+        });
+
+        return result.map((r: any) => r.toJSON().creator);
+    };
+
+    const findAllFollowingsByCreator = async (
+        creator: string,
+        offset = 0,
+        limit = 20,
+        order: 'DESC' | 'ASC' = 'DESC'
+    ): Promise<ConnectionModel[]> => {
+        let result = await model.findAll({
+            attributes: ['name'],
+            where: { creator },
+            offset,
+            limit,
+            order: [['createdAt', order]],
+        });
+
+        return result.map((r: any) => r.toJSON().name);
```

Comment:
let's return `messageId` instead of `name`

Reply:
table is like

|messageId (`creator/hash`)|hash|creator (user that follows)|name (user followed)|
|---|---|---|---|
|0x123/asdf|asdf|0x123|0xfoo|

`messageId` is always `creator/hash`,
If we return that,
I see how we can get the followers (we just extract the `creator` part from `messageId`),
but for the followings the info we need if `name` so I don't see how it can work (or we submit a second query to get a list of `name`s from a list of `hash`es?)